### ISSUE TITLE
state.bri_inc not yet implemented?

### ISSUE THREAD

**Issue by ebaauw** [dresden-elektronik/deconz-rest-plugin#84]
When I try and PUT /api/username/lights/1/state with a body of `{"bri_inc":254}` (or any other value), deCONZ returns nothing and the light's brightness doesn't change.  PUTting `{"ct_inc":512}` (or any other value) works as expected.  
Same for group requests.

**Comment by manup** [comment]
Correct, `ct_inc` was implemented to support IKEA tradfri remote color temperature buttons. The `bri_inc` is on the todo list.

... need better error messages for yet unsupported features.

**Comment by wvuyk** [comment]
same for hue_inc and sat_inc? Both were used in my plugin as well

**Comment by manup** [comment]
Yes currently only `ct_inc` is implemented, the others will follow but there is no ETA yet.

**Comment by ebaauw** [comment]
I started sniffing the commands the Hue bridge sends on PUTting these attributes, see https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Philips-Hue-sniffing.
